# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../dovecot/dovecot-2.2.1-0002-upstream-cac2978505b8.patch
# Copyright (C) 2013 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---


# HG changeset patch
# User Timo Sirainen <tss@iki.fi>
# Date 1366480686 -10800
# Node ID cac2978505b8f597a5f774821ff7217a4ad6209b
# Parent  46dc61f6acae8b441ec2b131fa10d6747f194dde
stats plugin: Fixed memory leak.

diff -r 46dc61f6acae -r cac2978505b8 src/plugins/stats/stats-plugin.c
--- a/src/plugins/stats/stats-plugin.c	Fri Apr 19 14:29:23 2013 +0300
+++ b/src/plugins/stats/stats-plugin.c	Sat Apr 20 20:58:06 2013 +0300
@@ -388,6 +388,7 @@
 
 	trans_stats_add(&suser->session_stats.trans_stats,
 			&strans->trans->stats);
+	i_free(strans);
 }
 
 static int

